sonar-plugin-api/src/main/java/org/sonar/api/utils/DateUtils.java

```diff
@@ -155,23 +156,34 @@ public static LocalDate parseLocalDateQuietly(@Nullable String s) {
   }
 
   /**
-   * @param s string in format {@link #DATETIME_FORMAT}
-   * @throws SonarException when string cannot be parsed
+   * @param s date time in ISO-8601 format. For historical reason, the format {@link #DATETIME_FORMAT} is supported
```

Comment:
If we want to stop supporting DATETIME_FORMAT someday, we should explain in the description that this format is deprecated starting 7.0 (we'll be able then to stop supporting it in 9.X)